### PR TITLE
Dynamic display of error page

### PR DESCRIPTION
**Pull request by Haroonchik**
First this PR merge https://github.com/HarshKapadia2/one-or-two/pull/19, then current.

### CODE REVIEW

**Comment by Haroonchik** [comment]
@KartikSoneji Check, I am ready

File: static/img/fist.svg [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+			S243.72,105.5,243.72,97.84z M195.94,81.91V26.57c0-3.49-2.84-6.33-6.33-6.33h-15.12c-3.49,0-6.33,2.84-6.33,6.33v55.34
+			c0,7.66,6.23,13.89,13.89,13.89S195.94,89.57,195.94,81.91z M148.17,81.91V26.57c0-3.49-2.84-6.33-6.33-6.33h-15.13
+			c-3.49,0-6.33,2.84-6.33,6.33v53.67c8.92,1.31,16.72,6.1,21.97,12.95C145.87,90.67,148.17,86.56,148.17,81.91z M129.35,113.76
+			c0-7.66-6.24-13.89-13.9-13.89H60.12c-3.49,0-6.33,2.84-6.33,6.33v15.12c0,3.49,2.84,6.33,6.33,6.33h55.33
+			C123.11,127.65,129.35,121.42,129.35,113.76z M100.39,79.88V26.57c0-3.49-2.84-6.33-6.33-6.33H78.94c-3.49,0-6.33,2.84-6.33,6.33
+			v53.31H100.39z"/>
+	</g>
+	<g>
+	</g>
+</g>
```
Comment (KartikSoneji) [inline static/img/fist.svg]: Please remove the extra `<g>` and `</g>` tags.

**Review by KartikSoneji — COMMENTED** [review]
Please remove the extra group tags from the svg file.
Everything else looks good.

**Comment by Haroonchik** [comment]
@KartikSoneji Check

**Comment by Haroonchik** [comment]
@HarshKapadia2 Check

**Comment by Haroonchik** [comment]
@HarshKapadia2 @KartikSoneji Is anyone here ?

**Pull request closed without merging**